internal/hud/view/view.go

```diff
@@ -38,11 +38,23 @@ type ViewState struct {
 	ShowNarration    bool
 	NarrationMessage string
 
-	// if non-0, which resource's log is currently shown in a modal (1-based index)
-	DisplayedLogNumber int
+	LogModal LogModal
 }
 
 type View struct {
+	Log       string
 	Resources []Resource
 	ViewState ViewState
 }
+
+type LogModal struct {
+	// if non-0, which resource's log is currently shown in a modal (1-based index)
```

Comment:
1-based index 😭

Reply:
yeahhhhhhhh
I originally had 0-based, and then -1 for "no log", but hit a few bugs where the struct hadn't been initialized so it was opening the log for the first resource, and decided to embrace the notion that the zero state of a struct should give reasonable default behavior